### PR TITLE
Add timetracker extension

### PR DESCRIPTION
**Pull request by lilic**

### CODE REVIEW

File: time_tracker_jsnjack@gmail.com/metadata.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "shell-version": ["3.16", "3.18", "3.20"],
+    "shell-version": ["3.16", "3.18", "3.20", "3.8"],
```
Comment (iaguis) [inline time_tracker_jsnjack@gmail.com/metadata.json]: Can you put `"3.8"` at the beginning to make the array sorted?

**Review by iaguis — COMMENTED** [review]
This looks good but somehow it doesn't work properly for me. I can't configure it by clicking on the timer on the panel.

I saw how it should work on Lili's computer but it doesn't work for me :/

File: time_tracker_jsnjack@gmail.com/metadata.json (hunk 1 above)
Comment (lilic) [reply time_tracker_jsnjack@gmail.com/metadata.json]: @iaguis fixed.

**Comment by lilic** [comment]
@iaguis yes I find that very odd, as I just now cleanly installed the extension and works fine. 😕 Maybe someone else can try?

**Review by iaguis — APPROVED** [review]
I installed it on a fresh coding image and it seems to work fine. Something must have been wrong with my VM. LGTM

**Pull request merged**